plugins/pencilblue/services/salesforce/salesforce_strategy_service.js

```diff
@@ -43,6 +43,19 @@ module.exports = function(pb) {
             }
         }
 
+        async register(req) {
+            try {
+                const settings = await this.getSalesforceSettings(req);
+                return {
+                    enableCustomRegister: settings.enable_custom_register_url,
+                    url: settings.custom_register_url
+                };
+            } catch (e) {
+                pb.log.error('Something went wrong during salesforce register: ', e);
+                return null;
```

Comment:
the salesforce error is too long, we don't need the whole error object. e.stack would be enough.